tests/testTriggers.go

```diff
@@ -10283,6 +10287,132 @@ func TriggerAsyncDRMigrationSchedule(contexts *[]*scheduler.Context, recordChan
 	})
 }
 
+func TriggerStorkVolumeSnapshotSchedule(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer endLongevityTest()
+	startLongevityTest(StorkVolumeSnapshotSchedule)
+	defer ginkgo.GinkgoRecover()
+	log.InfoD("Stork volume snapshot schedule triggered at: %v", time.Now())
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: StorkVolumeSnapshotSchedule,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+
+	var (
+		taskNamePrefix     = "snaptest"
+		snapshotType       = "cloud"
+		snapInterval = 2
+		retain storkapi.Retain = 3
+		scpolName = "snap-policy-" + time.Now().Format("15h03m05s")
+		snapNs []string
+	)
+
+	for i := 0; i < Inst().GlobalScaleFactor; i++ {
+		taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+		log.Infof("Task name %s\n", taskName)
+		appContexts := ScheduleApplications(taskName)
+		for _, ctx := range appContexts {
+			// Override default App readiness time out of 5 mins with 10 mins
+			ctx.ReadinessTimeout = appReadinessTimeout
+			namespace := GetAppNamespace(ctx, taskName)
+			snapNs = append(snapNs, namespace)
+		}
+		*contexts = append(*contexts, appContexts...)
+	}
+	ValidateApplications(*contexts)
+
+	schdPol, err := asyncdr.CreateSchedulePolicyWithRetain(scpolName, snapInterval, retain)
+	if err != nil {
+		UpdateOutcome(event, fmt.Errorf("Failed to create schedule policy"))
+		return
+	}
+
+	for _, ns := range snapNs {
+		pvcNames, err := GetPVCListForNamespace(ns)
+		if err != nil {
+			UpdateOutcome(event, fmt.Errorf("Failed to get pvc list for namespace"))
+			return
+		}
+		err = asyncdr.ValidateSnapshotScheduleCount(pvcNames, ns, schdPol.Name, snapshotType, snapInterval, retain)
+		if err != nil {
+			UpdateOutcome(event, fmt.Errorf("Failed to validate snapshot schedule count"))
+			return
+		}
+	}
```

Comment:
It would be helpful while debugging if we print params in the log messages.

Reply:
Added log for PVC and NS. Thanks!